### DISCUSSION TITLE
Terminal application with Spectre, is it possible?

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [spectreconsole/spectre.console#1580] [Q&A] [answered]
I am thinking about building a terminal application using dotnet, like a file manage MidnightCommander. Is Spectre capable of this? I haven't seen in the documentation hotkeys and other options.

**Comment by patriksvensson** [comment, marked as answer]
No, it's not designed for these kinds of scenarios. I would recommend taking a look at Gui.CS or similar frameworks.
